### DISCUSSION TITLE
Easy mode?

### DISCUSSION THREAD

**Discussion by OllieIz** [zippity21/Stardew_Valley_Regression_Mod#126] [Q&A]
I might be stupid, but what does easy mode do exactly?

**Comment by GardensStarlit** [comment]
As far as I can tell from a skim of the code, it _mostly_ just seems to remove/reduce the penalty for having an accident in bed. (Spot-washing the sheets, instead of fully washing them in the river)

It's possible that I've missed the other things, but I don't really get the point of it either in its current state.

**Comment by Gru1248** [comment]
It removes the need to eat and drink
